lib/node.js

```diff
@@ -468,7 +493,25 @@ declare module "punycode" {
 }
 
 declare module "querystring" {
-  // TODO
+  declare function stringify(
+    obj: Object,
+    separator?: string,
+    equal?: string,
+    options?: {
+      encodeURIComponent?: (str: string) => string;
+    }
+  ): string;
+  declare function parse(
+    str: string,
+    separator?: string,
```

Comment:
I think `separator` and `equal` probably want to be nullable (rather than optional).

The docs show examples where you might want to skip over one or both of these args to get to the options object -- so you can do that by specifying `null` in the params list.